pkg/controller/jobframework/tas.go

```diff
@@ -65,6 +70,17 @@ func NewPodSetTopologyRequest(meta *metav1.ObjectMeta) *podSetTopologyRequestBui
 		psTopologyReq = nil
 	}
 
+	if sliceRequiredTopologyFound {
+		if sliceRequiredTopologyFound {
+			psTopologyReq.PodSetSliceRequiredTopology = &sliceRequiredTopologyValue
+		}
+		if sliceSizeFound {
+			sliceSizeIntValue, _ := strconv.ParseInt(sliceSizeValue, 10, 32)
+			// TODO Error handling. For simplicity of reviewing a PR, it will be implemented in a follow-up
```

Comment:
Maybe we could add validation that checks the presence of this annotation

Reply:
I did that in this PR: https://github.com/kubernetes-sigs/kueue/pull/5582